docs/hydra/5min-tutorial.md

```diff
@@ -98,30 +102,6 @@ $ docker-compose -f quickstart.yml exec hydra \
     --client-id my-client \
     --client-secret secret
 
-UDYMha9TwsMBejEvKfnDOXkhgkLsnmUNYVQDklT5bD8.ZNpuNRC85erbIYDjPqhMwTinlvQmNTk_UvttcLQxFJY
```

Comment:
Please revert this change. The information shown here is correct and important.

Reply:
I dont't think the information is correct, since hydra does not display the token after the call - that's why I removed the token inspection command.

Reply:
The information is 100% correct, if it didn't show the token it was an error (such as misspelling, waiting too long before using the token, ...).